manual/core/metadata/schema/README.md

```diff
@@ -132,7 +132,53 @@ You can also limit the metadata to a subset of keyspaces:
 datastax-java-driver.advanced.metadata.schema.refreshed-keyspaces = [ "users", "products" ]
 ```
 
-If the property is absent or the list is empty, it is interpreted as "all keyspaces".
+Each element in the list can be one of the following:
+
+1. An exact name inclusion, for example `"Ks1"`. If the name is case-sensitive, it must appear in
+   its exact case.
+2. An exact name exclusion, for example `"!Ks1"`.
+3. A regex inclusion, enclosed in slashes, for example `"/^Ks.*/"`. The part between the slashes
```

Comment:
From your example it seems that the regex must match the entire input, not portions of it. Maybe it's worth mentioning this here.

Reply:
Actually no, `Pattern.matcher().matches()` also returns true for partial matches. I included the start symbol `^` because otherwise it will match stuff like `aaaKs`.

So even more justification for documenting it 👍

Reply:
> also returns true for partial matches

Hrmm, in fact no, it doesn't.

Reply:
Added a small note.